docs/guide/onboarding-library.md

```diff
@@ -1,9 +1,158 @@
 # Onboarding Library
+As an Ethereum enabled site developer, sending users offsite to install MetaMask presents challenges. Most notably, you must inform the user to return to your site and refresh their browser after the installation. Your site will detect the user's newly installed MetaMask extension only after that refresh. We at MetaMask care deeply about user experience, and we knew that this workflow needed to be improved.
 
-MetaMask now allows sites to register as onboarding the user, so that the user is redirected back to the initiating site after onboarding. This is accomplished through the use of the [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding).
+MetaMask now provides a [metamask-onboarding library](https://github.com/MetaMask/metamask-onboarding) designed to improve and simplify the onboarding experience. The new library exposes an API to initiate the onboarding process. In the process, it registers your site as the origin of the onboarding request. MetaMask will check for this origin after the user completes the onboarding flow. If it finds an origin, the final confirmation button of the MetaMask onboarding flow will indicate that the user will be redirected back to your site. The wording of this button helps prevent phishing attempts, by highlighting that a redirect is taking place to an untrusted third party.
 
-At the end of onboarding, the text of the button will show the origin of the site the user is about to be redirected to. This is intended to help prevent phishing attempts, as it highlights that a redirect is taking place to an untrusted third party.
+## Getting Started
+1. install @metamask/onboarding using npm or yarn.
+2. import the Onboarding Library or include it in your page.
 
+```javascript
+// As an ES6 module
+import MetaMaskOnboarding from '@metamask/onboarding';
+// Or as an ES5 module
+const MetamaskOnboarding = require('@metamask/onboarding');
+```
+
+If you'd prefer you can instead include the prebuilt ES5 bundle that ships with the library:
+```html
+  <script type="text/javascript" src="./metamask-onboarding.bundle.js"></script>
+```
+
+3. create a new instance of the Onboarding library
+
+```javascript
+const onboarding = new MetaMaskOnboarding();
+```
+
+4. kick off the onboarding process.
```

Comment:
We should make it clear that this should be in an event handler. If this is done upon page load, the browser will prevent the store tab from opening by default, and prompt the user to allow popups.

It needs to be in an event handler for a user action (e.g. a button click) to work reliably.

```suggestion
4. start the onboarding process in response to a user event (e.g. a button click)
```

Reply:
@Gudahtt was the removal of the period intentional? Should I remove them from the other bullets?

Reply:
Ah, sorry, that was an accident